packages/packagers/js/src/ScopeHoistingPackager.js

```diff
@@ -924,7 +924,13 @@ ${code}
     if (resolved.meta.isConstantModule) {
       invariant(
         this.bundle.hasAsset(resolved),
-        'Constant module not found in bundle',
+        `Constant module ${path.relative(
```

Comment:
This is probably fine as it only runs for constant modules, however I'd normally favour a `if(){throw err}` here as it means we only generate the error message when it's going to be thrown.

Reply:
Yeah, that's fair - ideally `message` could be a `() => string` function that's only invoked if the invariant fails (but then I guess the function is still created?)

Easy enough to change, I'll swap it over.